redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/DatabaseServerV4Endpoint.java

```diff
@@ -45,23 +43,22 @@
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = DatabaseServerOpDescription.LIST, produces = MediaType.APPLICATION_JSON, notes = Notes.DATABASE_SERVER_NOTES,
         nickname = "listDatabaseServers")
-    DatabaseServerV4Responses list(@NotNull @QueryParam("environmentId") String environmentId,
-        @QueryParam("attachGlobal") @DefaultValue("false") Boolean attachGlobal);
-
-    @GET
-    @Path("{nameOrCrn}")
-    @Produces(MediaType.APPLICATION_JSON)
-    @ApiOperation(value = DatabaseServerOpDescription.GET_BY_NAME_OR_CRN, produces = MediaType.APPLICATION_JSON, notes = Notes.DATABASE_SERVER_NOTES,
-            nickname = "getDatabaseServerByNameOrCrn")
-    DatabaseServerV4Response getByNameOrCrn(@NotNull @QueryParam("environmentId") String environmentId, @PathParam("nameOrCrn") String nameOrCrn);
+    DatabaseServerV4Responses list(@NotNull @QueryParam("environmentCrn") String environmentCrn);
 
     @GET
-    @Path("/crn/{crn}")
+    @Path("{crn}")
```

Comment:
Use the same pattern as the other module, for example: `EnvironmentEndpoint`

Reply:
Thanks.

First, EnvironmentEndpoint and e.g. SdxEndpoint use a somewhat different pattern. Namely, in sdx you have /{name} and /crn/{crn}.
Since in our case the main way to reach resources is via crn, we made the /{crn} the main way to get a resource and /name/{name the secondary. 

If you still think we should have /crn/{crn}, we can change.

Reply:
We did discuss the path format for CRN vs name for redbeams. We looked at the environment and freeipa services, and found that they have different standards. When I look right now, environment uses /crn and /name, but freeipa doesn't use either - environments passed by CRN are passed to endpoints without "/crn". (If I recall correctly, when we looked earlier, freeipa was using /crn but not /name.)

So, we decided to not use "/crn" because the CRN fits the definition of a RESTful resource identifier, and it's architecturally better to focus on it as the primary identifier in the API. So, "/crn" would not be necessary.